pegasus/forms/class_submission.rb

```diff
@@ -243,6 +243,7 @@ def self.query(params)
           4326
         )
       ) / 1000
+      query = query.where Sequel.lit(Forms.json('processed_data.location_p'))
```

Comment:
I find this syntax a bit hard to read. It might be nice to have a helper, something like `json_path_exists?(path)`.